packages/fxa-shared/metrics/amplitude.js

```diff
@@ -335,3 +336,13 @@ function mapNewsletterState (eventCategory, data) {
     return { newsletter_state };
   }
 }
+
+function mapNewsletters (data) {
+  let {newsletters} = data;
+  if (newsletters) {
+    newsletters = newsletters.map((newsletter) => {
+      return toSnakeCase(newsletter);
```

Comment:
@philbooth Do array values need to be snake case?

Reply:
They don't *need* to be, but if they are it will be more consistent with other values.